tests/components/zha/test_sensor.py

```diff
@@ -164,7 +164,7 @@ async def async_test_smart_energy_summation(hass, cluster, entity_id):
     await send_attributes_report(
         hass, cluster, {1025: 1, "current_summ_delivered": 12321, 1026: 100}
     )
-    assert_state(hass, entity_id, "12.32", UnitOfVolume.CUBIC_METERS)
+    assert_state(hass, entity_id, "12.321", UnitOfEnergy.KILO_WATT_HOUR)
```

Comment:
Why does the unit change?

Reply:
Due to this change https://github.com/home-assistant/core/pull/107685/commits/7d273e7f81df60d658599fad0aecf72c093fe978

Requested in https://github.com/home-assistant/core/pull/107685#discussion_r1451611113